dashboards/Experimental/Environment_Summary.json

```diff
@@ -8,16 +8,23 @@
         "hide": true,
         "iconColor": "rgba(0, 211, 255, 1)",
         "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
         "type": "dashboard"
       }
     ]
   },
   "description": "Detailed view on Environments",
   "editable": false,
+  "fiscalYearStartMonth": 0,
   "gnetId": 14372,
```

Comment:
strange that this dashboard has `gnetId` field when others don't

Reply:
> strange that this dashboard has `gnetId` field when others don't

Not really, there is plenty of those, for example: https://github.com/percona/grafana-dashboards/pull/1319/files#diff-b187478b147047e0d77293f487ca4283b7c84b2434b79cf072d2554bdd5f56fbR54

So do we need it, won't it break future upgrades?

Reply:
I would prefer to do it by book and set it to null.

Reply:
Done